integrations/gitlab/config.yaml

```diff
@@ -12,5 +12,5 @@ integration:
   type: "gitlab"
   # The configuration of the integration.
   config:
-    tokenMapping: {{ from env TOKEN_MAPPING }}
-    appHost: {{ from env APP_HOST }}
\ No newline at end of file
+    tokenMapping: \{\{ from env TOKEN_MAPPING \}\}
+    appHost: \{\{ from env APP_HOST \}\}
```

Comment:
if the escaping isn't a terraform thing but a yaml thing you could just wrap the value with `"`s